secure-api-gateway-ig-extensions/src/main/java/com/forgerock/sapi/gateway/common/jwt/ClaimsSetFacade.java

```diff
@@ -43,6 +47,18 @@ public ClaimsSetFacade(JwtClaimsSet claimsSet) {
         this.claimsSet = claimsSet;
     }
 
+    public boolean hasExpired() throws JwtException {
+        boolean hasExpired = false;
+        Date expirationTime = claimsSet.getExpirationTime();
+        if(expirationTime == null){
+            throw new JwtException("JWT has no expiration date");
+        }
+        if (expirationTime.before(new Date())) {
+            hasExpired = true;
+        }
+        return hasExpired;
```

Comment:
`return expirationTime.before(new Date())` 

Then you can remove the variable.